xchange-coindeal/src/main/java/org/knowm/xchange/coindeal/dto/trade/CoindealOrder.java

```diff
@@ -0,0 +1,96 @@
+package org.knowm.xchange.coindeal.dto.trade;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+public class CoindealOrder {
```

Comment:
you could use the value object patter (constructor + only getters) also for this object too to make it consistent with other DTOs

Reply:
and `final` members...

Reply:
@walec51 i have delete the setters now. @timmolter are you referring to make final the variables?

Reply:
Yes. See here: https://github.com/knowm/XChange/wiki/New-Implementation-Best-Practices#step-4

Reply:
I have done it today.